CONTRIBUTE.md

```diff
@@ -1,17 +1,23 @@
-#Contribution
+# Contribution
 If you want to contribute code to the frontend repository, you should first read through these guidelines.
 
-##General
+## Development process
+
+If you have an idea about a feature or a change to an existing feature, just open an issue in the frontend repo and describe your idea briefly. Members of the development team will look into it and review the idea. Then write some code and stay in contact with us. If you need additional features in the backend, open an issue there (and reference the main frontend issue).
+
+The feature will get merged if there is no resistance from the development team.
+
+## General
 * before committing make sure `npm run lint` and `npm test` were successful
 * branches and PRs instead of direct commits to master (except it is a small and urgent fix)
 * if you are working on issues create a branch labeled as what your change (in one word) postfixed with `#` and issue number
 * claim in the issue comments when you're working on one, and take part in the discussion
 * follow the style guide
```

Comment:
is it possible to put links/anchors in here? would be nice, if "follow the style guide" would be a link towards the Code Style section